controllers/statuscheck/manager.go

```diff
@@ -124,14 +124,6 @@ func (c *workerCache) delete(key types.NamespacedName) {
 	worker.stop()
 }
 
-func (c *workerCache) get(key types.NamespacedName) (*worker, bool) {
-	obj, ok := c.workers.Load(key)
-	if !ok {
-		return nil, ok
-	}
-	return obj.(*worker), ok
-}
-
```

Comment:
duplicated with #3228, I prefer to use that one because the changes of that PR are smaller and the description is more accurate.

Reply:
@STRRL I created this PR before that. Should I undo the changes? :)